ios/PhotoCompressor.mm

```diff
@@ -4,19 +4,23 @@
 @implementation PhotoCompressor
```

Comment:
Only now I've noticed that you are not launching compression in a different thread. Reimplement this to launch compression in a different thread